infra/testing/test-server.js

```diff
@@ -1,19 +1,51 @@
 const express = require('express');
 const oneLine = require('common-tags').oneLine;
+const path = require('path');
+const serveIndex = require('serve-index');
 
 const logHelper = require('../utils/log-helper');
 
 const app = express();
 
-app.get('/', (req, res) => {
-  res.send('Hello World!');
+app.get(/\/__WORKBOX\/buildFile\/(workbox-[A-z]*)(\.(?:dev|prod)\.(.*))*/, (req, res) => {
+  const moduleName = req.params[0];
+  const fileExtension = req.params[2];
+
+  const modulePath = path.join(__dirname, '..', '..', 'packages', moduleName);
+  const pkg = require(
+    path.join(modulePath, 'package.json')
+  );
+
+  const browserFile = pkg.browser;
+  const libraryPath = path.dirname(path.join(modulePath, browserFile));
+  let libraryFileName = path.basename(browserFile);
+  switch (process.env.NODE_ENV) {
+    case 'dev':
```

Comment:
Can you read this from https://github.com/GoogleChrome/workbox/blob/v3/gulp-tasks/utils/constants.js#L24 instead of hardcoding the possible values?

Reply:
Nope. it's `dev` and `prod` for file names instead of `dev` and `production` and even then not idea how I'd reference that in a case statement like this unless I changed structure to an object in constants which might fix both of these issues.

Reply:
Fine to keep as-is for now.

Let's chat about it in person, because I think we need to expose more info about the environment constants and library build names somewhere. I'm bumping up against this with `workbox-build` tests, too.

Reply:
See https://github.com/GoogleChrome/workbox/issues/851